.gitignore

```diff
@@ -1,5 +1,13 @@
+# Eclipse
 /.classpath
 /.project
 /.settings/
+
+# IntelliJ IDEA
+.idea
+*.iml
+
+# Build and tests
 /bin/
 /dist/
+/test-report/
```

Comment:
There is no test reports produced so far, I would be fine with merging the IntelliJ part though. The test results should be added to .gitignore with a PR that actually generates such results, so please remove it from this patch and add it to another PR accordingly.